src/pages/About/About.page.tsx

```diff
@@ -1,5 +1,23 @@
-const AboutPage = () => {
-  return <div>AboutPage</div>;
+'use client';
+
+import React from 'react';
+import FeaturesCards from './components/FeaturesCards/FeaturesCards';
+
+const AboutUs: React.FC = () => {
+  return (
+    <div style={{ padding: '2.5rem 1.25rem', maxWidth: '1200px', margin: '0 auto' }}>
+      <section style={{ textAlign: 'center', marginBottom: '3rem' }}>
+        <h1 style={{ fontSize: '2.5rem', fontWeight: '700', marginBottom: '1rem' }}>About Us</h1>
+        <p style={{ fontSize: '1.125rem', color: '#555' }}>
```

Comment:
Please use the existing typography and not tags like `p` or `h1`
No inline styles
Colors should come from the theme